model/metrics/clr-metrics.yaml

```diff
@@ -0,0 +1,169 @@
+groups:
+  - id: metric.clr.gc.collections.count
+    type: metric
+    metric_name: clr.gc.collections.count
+    brief: >
+      Number of garbage collections that have occurred since the process started.
```

Comment:
```suggestion
    brief: "Number of garbage collections that have occurred since the process started."
```

Reply:
In order to fix linter as a hotfix untel bug will be fixed you need to change all of yours briefs for metrics to be one liner. And run generate table again